CHANGELOG.md

```diff
@@ -3,6 +3,20 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## Windows/Linux 4.1.6 - 2017-02-03
+Updated PECL release packages. Here is the list of updates:
+### Added
+- Merge Linux and Windows code.
```

Comment:
One more thing, are we going to modify the version number in shared/version.h as well?